pkgs/by-name/li/licensed/package.nix

```diff
@@ -0,0 +1,19 @@
+{
+  lib,
+  bundlerApp,
+  bundlerUpdateScript,
+}:
+bundlerApp {
+  pname = "licensed";
+  gemdir = ./.;
+  exes = [ "licensed" ];
+
+  passthru.updateScript = bundlerUpdateScript "licensed";
+
+  meta = {
+    description = "A Ruby gem to cache and verify the licenses of dependencies";
```

Comment:
```suggestion
    description = "Ruby gem to cache and verify the licenses of dependencies";
```

Reply:
Ty. (I should have brought back what I learned from all my PRs since…)